kafka/1.10/README.md

```diff
@@ -0,0 +1,171 @@
+
+- Estimated time for completion: 20 minutes
+- Target audience: Anyone interested in a pub-sub messaging system.
+- Scope:
+ - Install the DC/OS Kafka service.
+ - Use the enhanced DC/OS CLI to create Kafka topics.
+ - Use Kafka on DC/OS to produce and consume messages.
+
+**Terminology**:
+
+- **Broker:** A Kafka message broker that routes messages to one or more topics.
+- **Topic:** A Kafka topic is message filtering mechanism in the pub/sub systems. Subscribers register to receive/consume messages from topics.
+- **Producer:** An application that producers messages to a Kafka topic.
+- **Consumer:** An application that consumes messages from a Kafka topic.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Kafka](#install-kafka)
+ - [Typical installation](#typical-installation)
+ - [Minimal installation](#minimal-installation)
+- [Topic Management](#topic-management)
+- [Produce and consume messages](#produce-and-consume-messages)
+ - [List Kafka client endpoints](#list-kafka-client-endpoints)
+ - [Produce a message](#produce-a-message)
+ - [Consume a message](#consume-a-message)
+- [Uninstall Kafka](#uninstall-kafka)
+
+## Prerequisites
+
+- A running DC/OS 1.10 cluster with 4 private agents each with 2 CPUs and 2 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.10/usage/cli/install/) installed.
+
+## Install Kafka
+
+Assuming you have a DC/OS cluster up and running, you have two options to install Kafka, described in the following.
+
+### Typical installation
+
+Install a Kafka cluster with 3 brokers using the DC/OS CLI:
+
+```bash
+$ dcos package install kafka
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
+Default configuration requires 3 agent nodes each with: 1.0 CPU | 2048 MB MEM | 1 5000 MB Disk
+Continue installing? [yes/no] yes
+Installing Marathon app for package [kafka] version [2.0.2-0.11.0]
+Installing CLI subcommand for package [kafka] version [2.0.2-0.11.0]
+New command available: dcos kafka
+The DC/OS Apache Kafka service is being installed.
+
+	Documentation: https://docs.mesosphere.com/service-docs/kafka/
+	Issues: https://docs.mesosphere.com/support/
+```
+
+Note that while the DC/OS CLI subcommand `kafka` is immediately available, it takes a few moments for Kafka to start running in the cluster.
+
+To validate that the Kafka service is running and healthy you can go to the DC/OS UI:
+
+![Services](img/services.png)
+
+### Minimal installation
+
+To start a minimal cluster with a single broker, create a JSON options file named `kafka-minimal.json`:

```

Comment:
We should add a comment that a simple `dcos package kafka` is sufficient as well